docs/modelling/configuring.rst

```diff
@@ -0,0 +1,263 @@
+
+For example, relational database achieve this by including the concurrency token in the ``WHERE`` clause of any ``UPDATE`` or ``DELETE`` commands and checking the number of rows that were affected. If the concurrency token still matches then one row will be updated. If the value in the database has changed, then no rows are updated.
+
+.. code-block:: sql
+
+    UPDATE [Person] SET [Name] = @p1
+    WHERE [PersonId] = @p0 AND [SocialSecurityNumber] = @p2;
+
+Ignoring Types and Properties
+-----------------------------
+
+EF uses conventions to discover the types and properties that should be included in your model. Including a type or property in your model means that it will be saved to and queried from the database.
+
+Ignore a type
+^^^^^^^^^^^^^
+
+The following code shows how to ignore an entire type from your model.
+
+.. literalinclude:: configuring/sample/EFModelling.Configuring.FluentAPI/Samples/IgnoreType.cs
+        :language: c#
+        :lines: 6-22
+        :emphasize-lines: 7
+        :linenos:
+
+Ignore a property
+^^^^^^^^^^^^^^^^^
+
+The following code shows how to ignore a property from a type.
+
+.. literalinclude:: configuring/sample/EFModelling.Configuring.FluentAPI/Samples/IgnoreProperty.cs
+        :language: c#
+        :lines: 6-23
+        :emphasize-lines: 7-8
+        :linenos:
+
+Relational database configuration
+---------------------------------
+
+The configuration in this section is applicable to relational databases in general. The extension methods shown here will become available when you install a relational database provider (due to the shared *EntityFramework.Relational* package).
+
+.. note::
+    Not all relational database will support all configuration shown here. For example, SQLite does not support the concept of schemas.
+
+Relational table
+^^^^^^^^^^^^^^^^
+
+The following code maps the ``Blog`` class to a ``blogs`` table in the database.
+
+.. literalinclude:: configuring/sample/EFModelling.Configuring.FluentAPI/Samples/Relational/Table.cs
+        :language: c#
+        :lines: 5-21
+        :emphasize-lines: 7-9
+        :linenos:
+
+You can also specify a scema that the table belongs to.
+
+.. literalinclude:: configuring/sample/EFModelling.Configuring.FluentAPI/Samples/Relational/TableAndSchema.cs
+        :language: c#
+        :lines: 11-13
+        :emphasize-lines: 3
+        :linenos:
+
+Relational column
+^^^^^^^^^^^^^^^^^
+
+The following code maps the ``Blog.BlogId`` property to a ``blog_id`` column in the database.
+
+.. literalinclude:: configuring/sample/EFModelling.Configuring.FluentAPI/Samples/Relational/Column.cs
+        :language: c#
+        :lines: 5-22
+        :emphasize-lines: 7-10
+        :linenos:
+
+Relational data type
+^^^^^^^^^^^^^^^^^^^^
+
+The following code specifies that the column for ``Blog.Url`` should use the ``varchar(200)`` datatype.
+
+.. note::
+    If you are targetting more than one relational provider with the same model then you probably 

```

Comment:
Yep... I think I never came back to finish that thought... removed the `.. note::` and put the following text after the code example below.

> If you are targetting more than one relational provider with the same model then you probably want to specify a data type for each provider rather than a global one to be used for all relational provider.

```
   modelBuilder.Entity<Blog>()
        .Property(b => b.Url)
        .ForSqlServer()
        .ColumnType("varchar(200)");
```